eng/common/cibuild.sh

```diff
@@ -13,4 +13,6 @@ while [[ -h $source ]]; do
 done
 scriptroot="$( cd -P "$( dirname "$source" )" && pwd )"
 
-. "$scriptroot/build.sh" --restore --build --test --pack --publish --ci $@
\ No newline at end of file
+# Run under sudo so we can set ulimit
+# See https://github.com/dotnet/core-eng/issues/14808
+sudo . "$scriptroot/build.sh" --restore --build --test --pack --publish --ci $@
```

Comment:
I don't think you can really change anything under the `eng/common` folder, as the next arcade update will overwrite whatever changes you made.

Reply:
This is correct, but you can do whatever you want in a thing outside eng/common before calling into it. 

I've independently synched with @ulisesh  about the underlying file limit problem and he's going to explore how we can get this onto the machines in a way that prevents these unfortunate workarounds.